src/components/templates/TalkShowStudio/components/Player/Player.scss

```diff
@@ -51,8 +59,15 @@
       user-select: none;
     }
 
-    &--on {
+    &--off {
+      visibility: visible;
       background: rgb(124, 70, 251);
     }
   }
+
+  &:hover {
+    .Player__buttons {
+      visibility: visible;
+    }
+  }
```

Comment:
`&:` styles should go before the `&__` styles as they relate to the main class. So this should be before `&__background`

Reply:
This is a duplicate of https://github.com/sparkletown/sparkle/pull/1653#discussion_r659516699